src/dbsync/tests/utils/string_helper_test.h

```diff
@@ -0,0 +1,13 @@
+#pragma once
```

Comment:
Please, use: 
#ifndef _STRING_HELPER_H
#define _STRING_HELPER_H
...
#endif // _STRING_HELPER_H

instead of #pragma once.